pkgs/by-name/xe/xevd/package.nix

```diff
@@ -47,6 +47,6 @@ stdenv.mkDerivation (finalAttrs: {
     pkgConfigModules = [ "xevd" ];
     maintainers = with lib.maintainers; [ jopejoe1 ];
     platforms = lib.platforms.all;
-    broken = !stdenv.hostPlatform.isx86 || stdenv.hostPlatform.isDarwin;
+    broken = !stdenv.hostPlatform.isx86 || !stdenv.cc.isGNU;
```

Comment:
A comment with the error and/or an upstream reference would be great.

Reply:
https://github.com/mpeg5/xeve/issues/117

and from reading the cmake file